app/views/centers/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h2>Details of <%= @center.name %></h2>
+
+<table border="1px">
+  <thead>
+  <tr>
+    <th>Manager Name</th>
+    <th>Email</th> 
+    <th>Contact Number</th>
+    <th>Address</th>  
+  </tr>
+  </thead>
+
+  <tbody>
+  <tr>
+    <td id="data"><%= @center.manager_name %></td>
```

Comment:
`id` is reserved for individual element in the DOM (document object model). What you could do is to add an `id` to your table. Then target it on the CSS